packages/core/src/components/tree/tree.tsx

```diff
@@ -63,6 +67,14 @@ export class Tree<T = {}> extends React.Component<ITreeProps<T>, {}> {
 
     private nodeRefs: { [nodeId: string]: HTMLElement } = {};
 
+    public nodeFromPath(path: number[]): ITreeNode<T> {
+        let treeNodes = this.props.contents;
+        for (let _i = 0; _i < path.length - 1; _i++) {
```

Comment:
call it `i` cuz it's used. leading `_` is convention for _unused but necessary_ variables.